examples/folium_vincent_markers.py

```diff
@@ -17,7 +17,9 @@
 # Binned wind speeds for NOAA 46050.
 bins = range(0, 13, 1)
 cuts = pd.cut(NOAA_46050['wind_speed_cwind (m/s)'], bins)
-ws_binned = pd.value_counts(cuts).reindex(cuts.levels)
+#ws_binned = pd.value_counts(cuts).reindex(cuts.values.levels) # cuts is a series, so values should be accessed
+# and also levels have been depreciated
```

Comment:
No need to comment it out. You may just remove and we can rely on git's history to check the changes.